packages/dcos-checks/buildinfo.json

```diff
@@ -0,0 +1,4 @@
+{
+	"state_directory": true,
+	"username": "dcos_checks"
```

Comment:
I don't think we should be specifying a username here since this package doesn't provide a running service/component, just a check binary (or binaries) that'll get called by the CLI or HTTP API.